product_docs/docs/pgd/6/reference/overview/basic-architecture.mdx

```diff
@@ -10,24 +10,24 @@ redirects:
 EDB Postgres Distributed (PGD) is a distributed database solution that extends PostgreSQL's capabilities, enabling highly available and fault-tolerant database deployments across multiple nodes. 
 PGD provides data distribution with advanced conflict management, data-loss protection, high availability up to five 9s, and throughput up to 5X faster than native logical replication.
 
-PGD is built on a multi-master foundation (Bi-Directional Replication, or BDR) which is then optimized for performance and availability through PGD Proxy. 
-You can run PGD without PGD Proxy if you need a custom deployment that better utilizes the multi-master functionality. When running without PGD Proxy, writes are distributed among the nodes and replicated to one another, and conflict resolution is relied on for maintaining consistency.
-This approach can be more efficient depending on your architectural needs. However, PGD Proxy ensures lower contention and conflict through the use of a write leader. For each proxy instance, a single endpoint addresses all the data nodes in a group, removing the need for clients to round robin multi-host connection strings. 
+PGD is built on a multi-master foundation (Bi-directional replication, or BDR) which is then optimized for performance and availability through Connection Manager. 
+You can run PGD without Connection Manager if you need a custom deployment better utilizing the multi-master functionality. When running without Connection Manager, writes are distributed amongst the nodes, replicated to one another, and conflict resolution is relied upon for maintaining consistency.
+This can be more efficient depending on your architectural needs, however, Connection Manager ensures lower contention and conflict through the use of a write leader.
```

Comment:
```suggestion
This can be more efficient depending on your architectural needs. However, Connection Manager ensures lower contention and conflict through the use of a write leader.
```